### PR TITLE
Create a PR template w security review

### PR DESCRIPTION
**Pull request by teallarson**
## Change description

> This PR adds a PR Template to this repo.  Relevant to our work with Vanta, it includes a section to acknowledge security impact.
> https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

## Related issues

Does this PR fix a Github Issue?

> n/a

### Development

- [n/a] Application changes have been tested appropriately
> This PR doesn't affect an application, only the PR process

### Impact

- [X] Code follows company security practices and guidelines

- [X] Security impact of change has been considered

- [X] Performance impact of change has been considered

- [X] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:
> No special considerations or impact.  Puts us into compliance for SOC2 when we add this to other repos.

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+### Security
+
+- [ ] Code follows company security practices and guidelines
+- [ ] Security impact of change has been considered
+  > Potential security impact (if any):
+
+### Code review
+
+- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
+- [ ] "Ready for review" label attached and reviewers assigned
```
Comment (evantahler) [inline .github/pull_request_template.md]: This probably isn't needed - we use "draft" PRs when not ready, and "real" PRs indicate that they are ready for review.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] Changes have been reviewed by at least one other contributor
```
Comment (evantahler) [inline .github/pull_request_template.md]: This probably isn't needed, as we are now blocking PRs from being merged unless they have more than one reviewer

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] Pull request linked to task tracker where applicable
```
Comment (evantahler) [inline .github/pull_request_template.md]: This probably isn't needed if we follow our branch naming conventions to start with the Pivotal Tracker story ID

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] Application changes have been tested thoroughly
+
+### Security
+
+- [ ] Code follows company security practices and guidelines
+- [ ] Security impact of change has been considered
+  > Potential security impact (if any):
+
+### Code review
+
```
Comment (evantahler) [inline .github/pull_request_template.md:29]: I'd like to see a checklist item for adding tags so the PR will be included in our release notes (bug, enhancement, etc)

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+Does this PR fix a Github Issue?
+
+> Fix [#1]()
+
+## Checklists
+
+### Development
+
+- [ ] Application changes have been tested thoroughly
```
Comment (evantahler) [inline .github/pull_request_template.md]: "thoroughly" is a weird word.  Our guidelines say that sometimes we don't need test too.  Perhaps "appropriately" is a better word?

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+### Security
```
Comment (evantahler) [inline .github/pull_request_template.md]: Probably called `Impact`, so below we can ask about security, performance, migrations, etc.

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+## Change description
+
+> Description here
+
+## Type of change
+
+- [ ] Bug fix (fixes an issue)
+- [ ] New feature (adds functionality)
```
Comment (evantahler) [inline .github/pull_request_template.md]: Maybe we don't need this because of the labels?

**Review by evantahler — APPROVED** [review]
👍 looks good!

Are you going to copy this into all of our other repos as well?

**Comment by teallarson** [comment]
Yep I plan on adding the same template to the other repos being monitored.

**Pull request merged**